packages/teams-js/src/public/interfaces.ts

```diff
@@ -1078,9 +1078,88 @@ export enum DevicePermission {
   Media = 'media',
 }
 
+/**
+ * @hidden
+ *
+ * @internal
+ * Limited to Microsoft-internal use
+ *
+ * @beta
+ */
+export interface AppEligibilityInformation {
```

Comment:
should these interfaces be moved to copilot.ts file? since this is public/interfaces.ts directory?